scylla_shard_aware_extension_integration_test.go

```diff
@@ -0,0 +1,79 @@
+//go:build all || ignore
+// +build all ignore
+
+package gocql
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+func TestSession_GetShardAwareRoutingInfo_Integration(t *testing.T) {
+	const (
+		keyspace = "gocql_scylla_shard_aware"
+		table    = "test_column_metadata"
+	)
+
+	// prepare
+	{
+		cluster := createCluster(func(cc *ClusterConfig) {
+			cc.PoolConfig.HostSelectionPolicy = TokenAwareHostPolicy(RoundRobinHostPolicy())
+		})
+
+		session, err := cluster.CreateSession()
+		if err != nil {
+			t.Fatalf("failed to create tsesion '%v'", err)
+		}
+		defer session.Close()
+
+		err = createTable(session, `DROP KEYSPACE IF EXISTS `+keyspace)
+		if err != nil {
+			panic(fmt.Sprintf("unable to drop keyspace: %v", err))
+		}
+
+		err = createTable(session, fmt.Sprintf(`CREATE KEYSPACE %s
+	WITH replication = {
+		'class' : 'SimpleStrategy',
```

Comment:
nit: we have a desire to deprecate SimpleStrategy, let's not add more tests with it.

Reply:
✅